### DISCUSSION TITLE
Creating delightful animations with GTK

### DISCUSSION THREAD

**Discussion by cassidyjames** [elementary/edw#3] [Ideas]
Speaker: **TBD, please comment if interested**

This talk gives an overview of animation primitives in GTK and Gtk.CSS that can be combined to create delightful animations, such as close buttons that roll in from off screen or buttons and icons that feel interactive and alive.

**Comment by cassidyjames** [comment]
This seems right up @alainm23 or @calo001's alley, if they're interested. :wink:

**Comment by hezral** [comment]
I second this, we need more focus on making beautiful apps on linux. there's still much to explore and Gtk is capable of this, even more so in Gtk4.
